pkg/mongo/testing/changestream.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testing
+
+import (
+	"context"
+	"fmt"
+
+	mongoclient "github.com/googleinterns/knative-source-mongodb/pkg/mongo"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+// TestChangeStream wraps the fake mongo.ChangeStream.
+type TestChangeStream struct {
+	Data TestCSData
+}
+
+// TestCSData is the data used to configure the test MongoDb ChangeStream.
+type TestCSData struct {
+	DecodeErr      error
+	NewChange      bson.M
+	NextFnExecuted bool
```

Comment:
I think you don't need to configure this from outside. I see you are always setting it to true... 
It can just be a private variable you use (i.e., name starts with lowercase)...

Reply:
true, I'll fix this!